2019-10-27 15:19:03,624 ERROR [main] org.sdk.EBISystem: Error connection to the database
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at org.core.EBIDatabase.connect(EBIDatabase.java:72)
	at org.core.EBINeutrinoSystemInit.Init(EBINeutrinoSystemInit.java:136)
	at org.core.EBINeutrinoSystemInit.<init>(EBINeutrinoSystemInit.java:42)
	at org.core.EBIMain.<init>(EBIMain.java:170)
	at org.core.EBIMain.main(EBIMain.java:69)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'localhost' (using password: YES)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:795)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:720)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:688)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:130)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:540)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:202)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1348)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:163)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:841)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:822)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:447)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:237)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at org.core.EBIDatabase.connect(EBIDatabase.java:68)
	... 4 more
2019-10-27 15:19:03,663 ERROR [main] org.sdk.gui.dialogs.EBIExceptionDialog: No database connection possible
Would you like to configure a connection?

2019-10-27 15:19:22,295 ERROR [AWT-EventQueue-0] org.sdk.gui.dialogs.EBIExceptionDialog: Connection is ok!
2019-10-27 15:19:23,757 ERROR [AWT-EventQueue-0] org.sdk.gui.dialogs.EBIExceptionDialog: Database connection data, are saved successfully!
2019-10-27 15:19:24,506 ERROR [AWT-EventQueue-0] org.sdk.gui.dialogs.EBIExceptionDialog: Would you like to import the EBI Neutrino R1 database schema?
2019-10-27 15:20:05,092 ERROR [AWT-EventQueue-0] org.sdk.gui.dialogs.EBIExceptionDialog: Inserisci un contatto!
2019-10-27 15:20:10,236 ERROR [ShowReportThread] org.sdk.EBISystem: null
 Cause :null

java.lang.NullPointerException
	at org.sdk.EBISystem.printStackTrace(EBISystem.java:711)
	at org.core.EBIReportSystem$4.run(EBIReportSystem.java:331)
	at java.lang.Thread.run(Thread.java:748)
2019-10-27 15:20:10,236 ERROR [ShowReportThread] org.sdk.gui.dialogs.EBIExceptionDialog: null
 Cause :null

2019-10-27 15:20:22,936 ERROR [AWT-EventQueue-0] org.sdk.gui.dialogs.EBIExceptionDialog: Vuoi chiudere il programma?
2019-10-27 17:30:54,410 ERROR [AWT-EventQueue-0] org.sdk.gui.dialogs.EBIExceptionDialog: Sono satati fatti dei cambiamenti, vuole salvarli?
2019-10-27 17:30:56,149 ERROR [AWT-EventQueue-0] org.sdk.gui.dialogs.EBIExceptionDialog: Vuoi chiudere il programma?
2019-10-27 17:36:10,424 ERROR [ShowReportThread] org.sdk.EBISystem: java.io.FileNotFoundException: /Users/francesco/NetBeansProjects/EBINeutrinoClient/EBINeutrino/resources/reports/meetingCustomerLetterReport.jasper
 Cause :java.io.FileNotFoundException: /Users/francesco/NetBeansProjects/EBINeutrinoClient/EBINeutrino/resources/reports/meetingCustomerLetterReport.jasper

net.sf.jasperreports.engine.JRException: java.io.FileNotFoundException: /Users/francesco/NetBeansProjects/EBINeutrinoClient/EBINeutrino/resources/reports/meetingCustomerLetterReport.jasper
	at org.sdk.EBISystem.printStackTrace(EBISystem.java:729)
	at org.core.EBIReportSystem$4.run(EBIReportSystem.java:342)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.FileNotFoundException: /Users/francesco/NetBeansProjects/EBINeutrinoClient/EBINeutrino/resources/reports/meetingCustomerLetterReport.jasper
	at net.sf.jasperreports.engine.util.JRLoader.loadObject(JRLoader.java:127)
	at net.sf.jasperreports.engine.util.JRLoader.loadObject(JRLoader.java:116)
	at net.sf.jasperreports.engine.JasperFillManager.getReportSource(JasperFillManager.java:1105)
	at net.sf.jasperreports.engine.JasperFillManager.fill(JasperFillManager.java:320)
	at net.sf.jasperreports.engine.JasperFillManager.fillReport(JasperFillManager.java:850)
	at org.core.EBIReportSystem$4.run(EBIReportSystem.java:321)
	... 1 more
2019-10-27 17:36:10,427 ERROR [ShowReportThread] org.sdk.gui.dialogs.EBIExceptionDialog: java.io.FileNotFoundException: /Users/francesco/NetBeansProjects/EBINeutrinoClient/EBINeutrino/resources/reports/meetingCustomerLetterReport.jasper
 Cause :java.io.FileNotFoundException: /Users/francesco/NetBeansProjects/EBINeutrinoClient/EBINeutrino/resources/reports/meetingCustomerLetterReport.jasper

2019-10-27 17:38:29,535 ERROR [AWT-EventQueue-0] org.sdk.EBISystem: Malformed URL: null, openView.groovy
 Cause :null

groovy.util.ResourceException: Malformed URL: null, openView.groovy
	at org.sdk.EBISystem.printStackTrace(EBISystem.java:729)
	at org.core.guiRenderer.EBIGUIRenderer$17.run(EBIGUIRenderer.java:1470)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
2019-10-27 17:38:29,536 ERROR [AWT-EventQueue-0] org.sdk.gui.dialogs.EBIExceptionDialog: Malformed URL: null, openView.groovy
 Cause :null

2019-10-27 17:41:20,326 ERROR [AWT-EventQueue-0] org.sdk.gui.dialogs.EBIExceptionDialog: Sono satati fatti dei cambiamenti, vuole salvarli?
2019-10-27 17:41:21,878 ERROR [AWT-EventQueue-0] org.sdk.gui.dialogs.EBIExceptionDialog: Vuoi chiudere il programma?
2019-10-27 17:47:53,586 ERROR [AWT-EventQueue-0] org.sdk.EBISystem: Cannot open URL: file:/Users/francesco/NetBeansProjects/EBINeutrinoClient/EBINeutrino/resources/views//Users/francesco/NetBeansProjects/EBINeutrinoClient/EBINeutrino/resources/openView.groovy
 Cause :null

groovy.util.ResourceException: Cannot open URL: file:/Users/francesco/NetBeansProjects/EBINeutrinoClient/EBINeutrino/resources/views//Users/francesco/NetBeansProjects/EBINeutrinoClient/EBINeutrino/resources/openView.groovy
	at org.sdk.EBISystem.printStackTrace(EBISystem.java:723)
	at org.core.guiRenderer.EBIGUIRenderer$17.run(EBIGUIRenderer.java:1481)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
2019-10-27 17:47:53,589 ERROR [AWT-EventQueue-0] org.sdk.gui.dialogs.EBIExceptionDialog: Cannot open URL: file:/Users/francesco/NetBeansProjects/EBINeutrinoClient/EBINeutrino/resources/views//Users/francesco/NetBeansProjects/EBINeutrinoClient/EBINeutrino/resources/openView.groovy
 Cause :null

